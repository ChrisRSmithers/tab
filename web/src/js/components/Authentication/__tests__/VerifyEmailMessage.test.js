/* eslint-env jest */

import React from 'react'
import {
  shallow
} from 'enzyme'
import toJson from 'enzyme-to-json'

describe('VerifyEmailMessage tests', function () {
  it('renders without error', function () {
    const VerifyEmailMessage = require('../VerifyEmailMessage').default
    shallow(
      <VerifyEmailMessage />
    )
  })

  it('matches expected snapshot', function () {
    const VerifyEmailMessage = require('../VerifyEmailMessage').default
    const wrapper = shallow(
      <VerifyEmailMessage />
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
