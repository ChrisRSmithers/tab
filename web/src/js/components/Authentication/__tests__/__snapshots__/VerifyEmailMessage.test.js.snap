// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerifyEmailMessage tests matches expected snapshot 1`] = `
<Paper
  circle={false}
  rounded={true}
  style={
    Object {
      "backgroundColor": "#FFF",
      "maxWidth": 400,
      "padding": 24,
    }
  }
  transitionEnabled={true}
  zDepth={1}
>
  <p>
    Please check your email to verify your account.
  </p>
  <span
    style={
      Object {
        "display": "flex",
        "justifyContent": "flex-end",
        "marginTop": 24,
      }
    }
  >
    <RaisedButton
      disabled={false}
      fullWidth={false}
      label="CANCEL"
      labelPosition="after"
      onClick={[Function]}
      primary={false}
      secondary={false}
      style={
        Object {
          "marginRight": 8,
        }
      }
    />
    <RaisedButton
      disabled={false}
      fullWidth={false}
      label="RESEND EMAIL"
      labelPosition="after"
      onClick={[Function]}
      primary={true}
      secondary={false}
    />
  </span>
</Paper>
`;
